README.md

```diff
@@ -1 +1,2 @@
 # news-explorer-frontend
```

Comment:
In ` README.md` you need to add some info about your project: technologies, how to download, how to start, how to deploy the app. And you need to add the link to your **deployed app**.